tests/test_autodoc.py

```diff
@@ -371,6 +371,18 @@ def test_resolve_links_in_text(self):
             ),
         )
 
+        self.assertEqual(
+            resolve_links_in_text(
+                "Link to [`transformers.BertModel.forward#input_ids`].",
+                transformers,
+                small_mapping,
+                page_info,
+            ),
+            (
+                "Link to [input_ids](/docs/transformers/main/en/model_doc/bert.html#transformers.BertModel.forward.input_ids)."
+            ),
+        )
+
```

Comment:
Would add the same test here with ``[`~transformers.BertModel.forward#input_ids`]`` to make sure it works too.